Marlin/thermistortable_70.h

```diff
@@ -20,64 +20,24 @@
  *
  */
 
+// This is the thermistor lookup table for the stock BQ Hephestos 2 100k thermistor.
+// The table was created on 29/12/2017 with an ambient temperature of 20C, the device
+// used to take the measurements was an ANENG AN8009 DMM with a K-type probe.
+
 // R25 = 100 kOhm, beta25 = 4100 K, 4.7 kOhm pull-up, bqh2 stock thermistor
 const short temptable_70[][2] PROGMEM = {
-  { OV(  22), 300 },
-  { OV(  24), 295 },
-  { OV(  25), 290 },
-  { OV(  27), 285 },
-  { OV(  29), 280 },
-  { OV(  32), 275 },
-  { OV(  34), 270 },
-  { OV(  37), 265 },
-  { OV(  40), 260 },
-  { OV(  43), 255 },
-  { OV(  46), 250 },
-  { OV(  50), 245 },
-  { OV(  54), 240 },
-  { OV(  59), 235 },
-  { OV(  64), 230 },
-  { OV(  70), 225 },
-  { OV(  76), 220 },
-  { OV(  83), 215 },
-  { OV(  90), 210 },
-  { OV(  99), 205 },
-  { OV( 108), 200 },
-  { OV( 118), 195 },
-  { OV( 129), 190 },
-  { OV( 141), 185 },
-  { OV( 154), 180 },
-  { OV( 169), 175 },
-  { OV( 185), 170 },
-  { OV( 203), 165 },
-  { OV( 222), 160 },
-  { OV( 243), 155 },
-  { OV( 266), 150 },
-  { OV( 290), 145 },
-  { OV( 317), 140 },
-  { OV( 346), 135 },
-  { OV( 376), 130 },
-  { OV( 408), 125 },
-  { OV( 442), 120 },
-  { OV( 477), 115 },
-  { OV( 513), 110 },
-  { OV( 551), 105 },
-  { OV( 588), 100 },
-  { OV( 626),  95 },
-  { OV( 663),  90 },
-  { OV( 699),  85 },
-  { OV( 735),  80 },
-  { OV( 768),  75 },
-  { OV( 800),  70 },
-  { OV( 829),  65 },
-  { OV( 856),  60 },
-  { OV( 881),  55 },
-  { OV( 903),  50 },
-  { OV( 922),  45 },
-  { OV( 939),  40 },
-  { OV( 954),  35 },
-  { OV( 966),  30 },
-  { OV( 977),  25 },
+  { OV(  18), 270 },
+  { OV(  27), 248 },
+  { OV(  34), 234 },
+  { OV(  45), 220 },
+  { OV(  61), 205 },
+  { OV(  86), 188 },
+  { OV( 123), 172 },
+  { OV( 420), 110 },
+  { OV( 590),  90 },
+  { OV( 845),  56 },
+
+  { OV( 970),  25 },
```

Comment:
Is this new thermistor table confirmed accurate?

Reply:
Using a sample rate of one..
I always had issues with temperatures using the BQH2,  the "normal" print temp for PLA was at 230 which is almost ABS range. So I decided to take a temp curve for the stock NTC and things are really different from what we have been using. I don't believe my NTC is broken, it's fairly consistent when I took the measurements.

All measurements were taken using a metallic probe force clamped into the heater block.